src/components/Chart/Chart.js

```diff
@@ -1,27 +1,31 @@
 // chart components
-import ChartBar from './ChartBar'
+import ChartBar from "./ChartBar";
 // styles
-import styles from './Chart.module.css'
+import styles from "./Chart.module.css";
 
-const Chart = props => {
-    // get array of the values from all 12 months in AlbumsChart.js with .map
-    const yearValues = Object.values(props.dataPoints)
-    // find biggest value across months with Math.max
-    const maxValue = Math.max(...yearValues)
+const Chart = ({ dataPoints }) => {

```

Comment:
instead of just importing `props` into your component, you can destructure and just take what you want. this should make components a little bit cleaner so that you dont have to write `props.whateverPropIWant` over and over